ui/app/controllers/jobs/index.js

```diff
@@ -191,10 +191,15 @@ export default class IndexController extends Controller.extend(
     Visible jobs are those that match the selected namespace and aren't children
     of periodic or parameterized jobs.
   */
-  @computed('model.jobs.@each.parent')
+  @computed('model.jobs.@each.parent', 'qpNamespace', 'store', 'storedJobs.[]')
   get visibleJobs() {
     if (!this.model || !this.model.jobs) return [];
-    return this.model.jobs
+    return this.store
+      .peekAll('job')
```

Comment:
`question`: Should we use `storedJobs` here so we don't call the store again? I'm also noticing we're declaring `storedJobs.[]` as a dependency here, however `storedJobs` isn't a reactive property. What the benefit of this?